### ISSUE TITLE
"READONLY You can't write against a read only replica" when trying to create a ticket

### ISSUE THREAD

**Issue by MoacirSchmidt** [googleforgames/open-match#1522]
**What happened**:
Receiving "READONLY You can't write against a read only replica" error when trying to create a ticket through CreateTicket front end calling

**How to reproduce it (as minimally and precisely as possible)**:
Intermitent and no related to codebase. It seems related to REDIS default setup

**Anything else we need to know?**:
Sometimes after a restart it seem that openmatch is trying to connect to slave instead of master to create tickets. I would link to know if there is some config to force connect to redis master

**Output of `kubectl version`**:
```
~$ minikube kubectl version
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.3", GitCommit:"434bfd82814af038ad94d62ebe59b133fcb50506", GitTreeState:"clean", BuildDate:"2022-10-12T10:57:26Z", GoVersion:"go1.19.2", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v4.5.7
Server Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.3", GitCommit:"434bfd82814af038ad94d62ebe59b133fcb50506", GitTreeState:"clean", BuildDate:"2022-10-12T10:49:09Z", GoVersion:"go1.19.2", Compiler:"gc", Platform:"linux/amd64"}

```

**Cloud Provider/Platform (AKS, GKE, Minikube etc.)**: 
Minikube

**Open Match Release Version**:
v1.6.0

**Install Method(yaml/helm)**:
yaml

**Comment by mridulji** [comment]
Hey @MoacirSchmidt, May I know which mode of redis setup you are using out of cluster and sentinel?

**Comment by MoacirSchmidt** [comment]
Hi @mridulji thanks for your answer!

All setup I did from standard installation procedure: https://open-match.dev/site/docs/installation/yaml/

```
# Install the core Open Match services.
kubectl apply --namespace open-match \
  -f https://open-match.dev/install/v1.6.0/yaml/01-open-match-core.yaml
```  

After that I run "Install the Default Evaluator" procedure

```
kubectl apply --namespace open-match \
  -f https://open-match.dev/install/v1.6.0/yaml/06-open-match-override-configmap.yaml \
  -f https://open-match.dev/install/v1.6.0/yaml/07-open-match-default-evaluator.yaml
  
```

Nothing different from standard open-match setup

**Comment by mridulji** [comment]
Thanks for confirming, I tried twice but not able to reproduce such an issue on GKE or minikube. Would you please try again to see it persists?

**Comment by MoacirSchmidt** [comment]
Ok. I will try ASAP. Meanwhile I found a possibly related issue:

https://github.com/googleforgames/open-match/issues/1418#issuecomment-1017138219

**Comment by mridulji** [comment]
Yep, it was resolved by updating redis helm chart version from 12.x to 16.3 and works fine after that.
> Meanwhile I found a possibly related issue

**Comment by mridulji** [comment]
Hey @MoacirSchmidt, please let me know if this issue is resolved?

**Comment by MoacirSchmidt** [comment]
Sure! I will update server this week. Currently production server was reconfigured to only one REDIS instance to workaround

**Comment by mridulji** [comment]
Thanks for your reply, Please feel free to re-open this issue in case still facing the problem.
